torch_np/tests/numpy_tests/lib/test_function_base.py

```diff
@@ -130,6 +130,7 @@ def test_axes(self):
         assert_raises(np.AxisError, np.flip, np.ones((4, 4)), axis=-3)
         assert_raises(np.AxisError, np.flip, np.ones((4, 4)), axis=(0, 3))
 
+    @pytest.mark.xfail(reason='no [::-1] indexing')
```

Comment:
More than fair. You can even skip these I'd say, as there's no hope PyTorch will support this I don't think.